README.md

```diff
@@ -39,6 +39,24 @@ This document will walk you through getting up and running with the .NET SDK Sam
 1. Setting up build environment
 2. Setting up a vSphere test environment
 
+### Importing, building and running the Project Template 
+
+* To import the VMware Project Template, install the vsix file by double click to the file "Vmware-SDK-Automation-Sample-Template.vsix" located in "vsphere-automation-sdk-.net\Project Template". 
+
+1. Open Visual Studio
+2.  In the menu select File --> New --> Project...
+3. Select the template as the image below shows
+![New Project](screenshots/newProject.jpg?raw=true)
+4. Then right  click on the console project on the solution explorer and on the debug options add the "Command line arguments"
```

Comment:
Instructions 4,5,6,7 not required, since all we need the template is for users to import and then modify it as per their own requirements for the sample they will write.

Reply:
Agreed